packages/blocks/src/api/raw-handling/test/paste-handler.js

```diff
@@ -7,58 +7,52 @@ import { pasteHandler } from '@wordpress/blocks';
+			<th>Header Cell</th>
+			<th rowspan="2">Rowspan 2</th>
+		</tr>
+		<tr>
+			<th colspan="2">Colspan 2</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td>Body Cell</td>
+			<td>Body Cell</td>
+			<td rowspan="2">Rowspan 2</td>
+		</tr>
+		<tr>
+			<td colspan="2">Colspan 2</td>
+		</tr>
+	</tbody>
+	<tfoot>
+		<tr>
+			<td>Footer Cell</td>
+			<td>Footer Cell</td>
+			<td rowspan="2">Rowspan 2</td>
+		</tr>
+		<tr>
+			<td colspan="2">Colspan 2</td>
+		</tr>
+	</tfoot>
 </table>`;
 
-const googleDocsTableWithColspan = `
-<meta charset="utf-8"><b style="font-weight:normal;" id="docs-internal-guid-b0f68bdd-7fff-a054-94d1-43c2fdedca2a">
-    <div dir="ltr" style="margin-left:0pt;" align="left">
-        <table style="border:none;border-collapse:collapse;">
-            <colgroup>
-                <col width="185"/>
-                <col width="439"/>
-            </colgroup>
-            <tbody>
-            <tr style="height:21pt">
-                <td colspan="2"
-                    style="border-left:solid #000000 1pt;border-right:solid #000000 1pt;border-bottom:solid #000000 1pt;border-top:solid #000000 1pt;vertical-align:top;padding:5pt 5pt 5pt 5pt;overflow:hidden;overflow-wrap:break-word;">
-                    <p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span
-                            style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Test colspan</span>
-                    </p></td>
-            </tr>
-            </tbody>
-        </table>
-    </div>
-    <br/><br/></b>
+const googleDocsTableWithColspanAndRowspan = `
+<google-sheets-html-origin><style type="text/css"><!--td {border: 1px solid #cccccc;}br {mso-data-placement:same-cell;}--></style><table xmlns="http://www.w3.org/1999/xhtml" cellspacing="0" cellpadding="0" dir="ltr" border="1" style="table-layout:fixed;font-size:10pt;font-family:Arial;width:0px;border-collapse:collapse;border:none"><colgroup><col width="100"/><col width="100"/><col width="100"/></colgroup><tbody><tr style="height:21px;"><td style="overflow:hidden;padding:2px 3px 2px 3px;vertical-align:bottom;" data-sheets-value="{&quot;1&quot;:2,&quot;2&quot;:&quot;Body Cell&quot;}">Body Cell</td><td style="overflow:hidden;padding:2px 3px 2px 3px;vertical-align:bottom;" data-sheets-value="{&quot;1&quot;:2,&quot;2&quot;:&quot;Body Cell&quot;}">Body Cell</td><td style="overflow:hidden;padding:2px 3px 2px 3px;vertical-align:bottom;" rowspan="2" colspan="1" data-sheets-value="{&quot;1&quot;:2,&quot;2&quot;:&quot;Rowspan 2&quot;}"><span><div style="max-height:42px">Rowspan 2</div></span></td></tr><tr style="height:21px;"><td style="overflow:hidden;padding:2px 3px 2px 3px;vertical-align:bottom;" rowspan="1" colspan="2" data-sheets-value="{&quot;1&quot;:2,&quot;2&quot;:&quot;Colspan 2&quot;}">Colspan 2</td></tr></tbody></table>
```

Comment:
I don't know how the string was generated before the change, but this PR uses the HTML shown in the browser console.

![console](https://user-images.githubusercontent.com/54422211/208287117-9aab94dd-ddfd-43a3-a037-673fbaba9c3f.png)

Reply:
Could we restore the original string and add yours as a second test case?

Reply:
I understand! But I'm concerned about the different structure of the original string and the string I added.

@mpkelly 
How was the original string generated?

Reply:
@t-hamano, I ran the test with a response I knew wouldn't match and then copied the actual output from the Jest test run and used that. after reviewing that it looked as expected.